src/client/utils/validations.js

```diff
@@ -468,7 +475,6 @@ export {
   isValidMonetaryValue,
   isValidPhone,
   isValidEmail,
-  isValidAddress,
```

Comment:
Oh. Just realized `isValidAddressField` here isn't even being used at all? There's a different function defined in `Address` called `validateAddressField` that's actually being used.